lib/util/jshint-reporter.js

```diff
@@ -64,12 +54,22 @@ function getJsHintReporter(reporterName) {
   return jsHintReporter;
 }
 
+var yargsOptionDefiniton = {
+  key: 'reporter',
+  value: {
+    describe: 'Specify a custom JsHint reporter to use. '+
+      'Either a locally npm installed module, or an asolute path to one.',
+    alias: ['r'],
+    default: defaultReporterName,
+    string: true,
+  }
+};
 var checkJsHintReporter = yargs.createCheck()
   .withGate(function (argv) {
     return !argv.help;
   })
   .withTest({
-    reporter: function(value) {
+    'reporter': function(value) {
       if (typeof value === 'undefined') {
```

Comment:
- `value` will be undefined when the CLI option is unspecified
- `undefined` is mutable - https://gist.github.com/easierbycode/1953667